api/handlers.go

```diff
@@ -39,16 +40,31 @@ type (
 		Storage           int64  `json:"storageLimit"`
 	}
 
-	// userUpdateData defines the fields of the User record that can be changed
+	// accountRecoveryDTO defines the payload we expect when a user is trying to
+	// change their password.
+	accountRecoveryDTO struct {
```

Comment:
What does `DTO` stand for? In skyd we would just call this `accountRecoveryPOST`.

Reply:
It's a standard abbreviation: https://en.wikipedia.org/wiki/Data_transfer_object

Reply:
Let's pls stick to our skyd code conventions. DTO is very generic and really doesn't give any context about the usage of the object. Suffixes like `GET` or `Request`/`Response` give a bit more context even if they are just DTOs. The latter is also a bit more friendly than an abbrevation.

Reply:
I really don't like this because it replaces an industry-standard term with a bunch of in-house ones which are also not very informative. Calling something DTO means that it's a wrapper used only for transportation and the object should not be used for anything else. It also means that the object doesn't have any functionality attached to, aside from formatting helpers.

I can see the benefit of sticking to conventions, though. So, sure, I'll rename all DTOs to <verb>/request/response types.

Reply:
> I can see the benefit of sticking to conventions, though.

This is really the only reason this should be renamed. DTO was the correct term here imo. DTOs are pretty widely adopted, if you've worked with ORMs or frameworks like symfony or react or whatever, DTO as a concept is baked into all of those.

Reply:
All DTOs renamed.